webpack.config.js

```diff
@@ -77,7 +77,9 @@ module.exports = {
       },
     }),
   ],
+  devtool: 'source-map',
```

Comment:
sourcemaps are disabled by default.  we can debate on [which devtool we want to use](https://webpack.js.org/configuration/devtool/), but I figured for now I'd just get something in here.